samples/client/petstore/elm-0.18/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-3.3.2-SNAPSHOT
\ No newline at end of file
+3.3.4-SNAPSHOT
```

Comment:
The renaming of `encode` will be a breaking change for me (no problem though). Don't know about versioning in this project, but if this is the version of the whole openapi generator-project, then it's probably fine :)

Reply:
3.3.4 will be the next release, hence the change.

Reply:
Due to the renaming of `encode`, I would suggest targeting this change for 4.0.x branch (v4.0.0 will be released in Dec 20th)

Reply:
Typically you do not need to use the `encode` directly. But if this is considered a 'Breaking change (without fallback)' I'll target it for 4.0.x.